src/Auth.js

```diff
@@ -1,16 +1,40 @@
+// Libraries
 import React from "react";
 import TwitterLogin from "react-twitter-login";
+import { withRouter } from "react-router-dom";
 
-const Auth = ({ addUser }) => {
-  const authHandler = (error, data) => {
+import { API, graphqlOperation } from "aws-amplify";
+import { createUser } from "./graphql/mutations";
+
+const Auth = ({ addUser, history }) => {
+  const authHandler = async (error, data) => {
     if (error) {
-      alert("Please log into Twitter!");
+      alert(error);
```

Comment:
Could you swap the alert back?

Reply:
Done!

Just pushed the change onto this same branch. It should appear below as commit [4ff1fe3](https://github.com/MLH-Fellowship/0.2.1-donation-trigger/pull/26/commits/4ff1fe300c532f58b7073badf65a104eecadfcc8)